Makefile

```diff
@@ -11,23 +11,22 @@ help:
 
 builds:
 	@echo "Building Docker Images... 🐳"
-	docker-compose build --no-cache
+	docker-compose build
```

Comment:
Do you just enjoy pain, or....? :sweat_smile:

Reply:
hahaha, good point!